package.json

```diff
@@ -131,7 +131,7 @@
     "postcss": "^8.2.1",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
-    "react-intl": "^5.10.6",
+    "react-intl": "^2.9.0",
```

Comment:
In https://github.com/cerner/terra-toolkit/pull/811 react-intl and main, both are still using v5, do we need to downgrade?

Reply:
That was an accidental change, good catch! It's been reverted in [4028db0](https://github.com/cerner/terra-toolkit/pull/812/commits/4028db0bebb805f3e3cc9c0d8b5d807b3a47a5a3)